Source/Core/DolphinQt/MemoryEngine/GUICommon.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <QString>
+#include <QStringList>
+
+#include "../../MemoryEngine/Common/MemoryCommon.h"
```

Comment:
We generally try to include our own files realtive to the root of the given project.

This should likely be
```suggestion
#include "DolphinQt/MemoryEngine/Common/MemoryCommon.h"
```

Same for pretty much all the other locations as well.